webapp/src/dogma/features/auth/authSlice.ts

```diff
@@ -16,130 +16,142 @@
 
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import axios from 'axios';
-import qs from 'qs';
 import { UserDto } from 'dogma/features/auth/UserDto';
-import { HttpStatusCode } from 'dogma/features/api/HttpStatusCode';
-import { setSessionId, removeSessionId, getSessionId } from 'dogma/features/auth/util';
 
-const getUser = createAsyncThunk('/auth/user', async () => {
-  // TODO(ikhoon): Just use fetch API?
-  const response = await axios.get(`${process.env.NEXT_PUBLIC_HOST}/api/v0/users/me`);
-  return response.data as UserDto;
+export const getUser = createAsyncThunk('/auth/user', async (_, { getState, rejectWithValue }) => {
+  try {
+    const { auth } = getState() as { auth: AuthState };
+    if (!auth.sessionId) {
+      return rejectWithValue('Session id not provided');
+    }
+    const { data } = await axios.get(`${process.env.NEXT_PUBLIC_HOST}/api/v0/users/me`, {
```

Comment:
How about adding a const value in a network utility class to return the hostname of the API server?
```
const DOGMA_HOST = process.env.NEXT_PUBLIC_HOST || "";
```

Reply:
Updated the axios config instead.